build/Gulpfile.js

```diff
@@ -12,7 +12,7 @@ const bump = require('gulp-bump');
 const path = require('path');
 const run = require('gulp-run');
 
-const keyStorePassword = require('./keystore.json').password;
+const keyStorePassword = require('../keystore.json').password;
```

Comment:
@borisyankov added just to test. Its a mistake 😉 The path issue should be resolved now btw. :)